schedule/yast/sles+sdk+proxy_SCC_via_YaST_Online_pvm.yaml

```diff
@@ -0,0 +1,33 @@
+---
+name: sles+sdk+proxy_SCC_via_YaST
+description: >
+  Add add-on via YaST control center directly https://progress.opensuse.org/issues/16402
+vars:
+  DESKTOP: gnome
+  SCC_ADDONS: sdk
+  SCC_REGISTER: yast
+  YUI_REST_API: 1
+schedule:
+  - installation/bootloader_start
+  - installation/setup_libyui
+  - installation/product_selection/install_SLES
+  - installation/licensing/accept_license
+  - installation/registration/register_via_scc
+  - installation/module_registration/register_module_desktop
+  - installation/add_on_product/skip_install_addons
+  - installation/system_role/accept_selected_role_SLES_with_GNOME
```

Comment:
then here we need to use:
`installation/system_role/accept_selected_role_text_mode`

Reply:
> then here we need to use: `installation/system_role/accept_selected_role_text_mode`

I have a question here, the original case is tested with gnome role, need to keep it or change to role of text mode please?
The original case: https://openqa.suse.de/tests/8752091, this case is started after test "create_hdd_gnome", the system role of the system is gnome.

Reply:
powerVM should not be tested with any desktop, original test case is wrong unfortunately. Only I hope that changing the role we can run fine the last test module. So we can try and see.